### PR TITLE
[🔧 ESLint] fixes for `no-implicit-coercion`

### PR DESCRIPTION
**Pull request by B2o5T**

### CODE REVIEW

File: packages/services/api/src/modules/schema/providers/schema-publisher.ts [hunk 1]
```diff
@@ -494,15 +494,14 @@ export class SchemaPublisher {
...
-      !!previousSchema &&
+      Boolean(latest.version && previousSchema) &&
       (previousSchema.url ?? null) !== (incomingSchema.url ?? null);
     const hasSchemaChanges = changes.length > 0;
     const hasErrors = errors.length > 0;
     const isForced = input.force === true;
     let hasDifferentChecksum = false;
 
-    if (!!latest.version && !!previousSchema) {
+    if (Boolean(latest.version) && Boolean(previousSchema)) {
```
Comment (B2o5T) [inline packages/services/api/src/modules/schema/providers/schema-publisher.ts:504]:
```suggestion
    if (latest.version && previousSchema) {
```
no need here

File: packages/web/app/pages/[orgId]/subscription/index.tsx [hunk 2]
```diff
@@ -128,11 +128,11 @@ export const getServerSideProps = withSessionProtection(async context => {
    */
   const isStripeEnabled = getIsStripeEnabled();
   if (isStripeEnabled === false) {
-    const parts = `${context.resolvedUrl}`.split('/');
+    const parts = String(context.resolvedUrl).split('/');
     parts.pop();
     return {
       redirect: {
-        destination: `${parts.join('/')}`,
+        destination: parts.join('/'),
```
Comment (B2o5T) [inline packages/web/app/pages/[orgId]/subscription/index.tsx:135]: here also, join returns string

**Review by n1ru4l — APPROVED** [review]

**Comment by n1ru4l** [comment]
@B2o5T There seems to be a lint issue

**Review by saihaj — APPROVED** [review]

File: packages/services/api/src/modules/schema/providers/schema-publisher.ts [hunk 3]
```diff
@@ -498,8 +498,8 @@ export class SchemaPublisher {
     const hasPreviousVersion = 'version' in latest && !!latest.version;
     const hasNewUrl =
       hasPreviousVersion &&
-      !!previousSchema &&
-      (previousSchema.url ?? null) !== (incomingSchema.url ?? null);
+      Boolean(previousSchema) &&
+      (previousSchema!.url ?? null) !== (incomingSchema.url ?? null);
```
Comment (n1ru4l) [inline packages/services/api/src/modules/schema/providers/schema-publisher.ts:502]:
```suggestion
      previousSchema != null &&
      (previousSchema.url ?? null) !== (incomingSchema.url ?? null);
```

**Review by dotansimha — APPROVED** [review]

**Comment by dotansimha** [comment]
@kamilkisiela can we merge?

File: packages/libraries/cli/src/commands/operations/publish.ts [hunk 4]
```diff
@@ -52,7 +52,7 @@ export default class OperationsPublish extends Command {
         normalize: true,
       });
       const collectedOperationsTotal = operations.length;
-      const noMissingHashes = operations.some(op => !!op.operationHash);
+      const noMissingHashes = operations.some(op => Boolean(op.operationHash));
```
Comment (B2o5T) [inline packages/libraries/cli/src/commands/operations/publish.ts:55]:
```suggestion
      const noMissingHashes = operations.some(op => op.operationHash);
```

File: packages/web/app/pages/manage/index.tsx [hunk 5]
```diff
@@ -77,8 +77,8 @@ function Manage() {
               <CheckboxGroup
                 colorScheme="teal"
                 size="sm"
-                defaultValue={Object.keys(filters).filter(
-                  key => !!filters[key as keyof typeof filters],
+                defaultValue={Object.keys(filters).filter(key =>
+                  Boolean(filters[key as keyof typeof filters]),
```
Comment (B2o5T) [inline packages/web/app/pages/manage/index.tsx:81]:
```suggestion
                 filters[key as keyof typeof filters],
```

File: scripts/sync-env-files.js [hunk 6]
```diff
@@ -8,7 +8,7 @@ import { dirname, join, relative } from 'path';
 import { parse } from 'dotenv';
 import glob from 'glob';
 
-if (!!process.env.CI) {
+if (Boolean(process.env.CI)) {
```
Comment (B2o5T) [inline scripts/sync-env-files.js:11]:
```suggestion
if (process.env.CI) {
```

**Pull request merged**